tests/Algorithms/HungarianAlgorithmTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Bearded.Utilities.Algorithms;
+using FsCheck.Xunit;
+using Xunit;
+
+namespace Bearded.Utilities.Tests.Algorithms
+{
+    public class HungarianAlgorithmTests
+    {
+        [Fact]
+        public void Run_EmptyCostMatrix_DoesntFail()
+        {
+            Assert.Empty(HungarianAlgorithm.Run(new float[0, 0]));
```

Comment:
`DoesntFail` is not the same as checking `Empty`, is it? Should be renamed, changed, or split into two tests. Also please avoid contractions.